packages/liveblocks-react/src/liveblocks.tsx

```diff
@@ -384,10 +361,16 @@ function makeLiveblocksContextBundle<
 }
 
 function useInboxNotifications_withClient(client: OpaqueClient) {
-  const { store, useSubscribeToInboxNotificationsEffect } =
+  const { loadInboxNotifications, store, useEnableInboxNotificationsPolling } =
     getExtrasForClient(client);
 
-  useSubscribeToInboxNotificationsEffect();
+  // Trigger initial loading of inbox notifications if it hasn't started
+  // already, but don't await its promise.
+  useEffect(() => {
+    loadInboxNotifications();
+  }, [loadInboxNotifications]);
```

Comment:
Since, this hook `useInboxNotifications_withClient` will also be called from `useInboxNotificationsSuspense_withClient` hook, this will mean suspense hook will perform data fetching twice? - first during render and second during this `useEffect` callback.

Reply:
It will indeed _trigger_ twice, but since the value of the promise is memorized after the first attempt, it will not actually fetch twice.

But you’re right: the moment of first trigger is different between the Suspense and non-Suspense versions:
- Suspense version will first trigger during render (because of the `use(…)`)
- Non-Suspense will first trigger in a `useEffect` (so after mounting)